libs/features/dynamic-navigation/dynamic-navigation.js

```diff
@@ -27,6 +27,13 @@ export default function dynamicNav(url, key) {
   const metadataContent = getMetadata('dynamic-nav');
   const dynamicNavGroup = getMetadata('dynamic-nav-group');
 
+  if (metadataContent === 'reset') {
+    window.sessionStorage.removeItem('gnavSource', url);
+    window.sessionStorage.removeItem('dynamicNavKey', key);
+    window.sessionStorage.removeItem('dynamicNavGroup', dynamicNavGroup);
```

Comment:
Nit: Do you want to add the keys as variables if you're using them more than once?